pynest/nest/lib/hl_api_info.py

```diff
@@ -55,23 +58,23 @@ def authors():
 
 
 @check_stack
-def helpdesk(browser="firefox"):
-    """Open the NEST helpdesk in the given browser.
+def helpdesk():
+    """Open the NEST helpdesk in browser.
 
-    The default browser is firefox.
-
-    Parameters
-    ----------
-    browser : str, optional
-        Name of the browser to use
+    The system default browser.
+    /helpdesk << /command (firefox) >> SetOptions in ~/.nestrc is now obsolete
     """
-
-    sr("/helpdesk << /command (%s) >> SetOptions" % browser)
-    sr("helpdesk")
+    url = os.path.join(os.environ['NEST_DOC_DIR'] + "/help", "helpindex.html")
```

Comment:
Provide a useful error message if `NEST_DOC_DIR` is undefined, similar as for missing `NEST_INSTALL_DIR`.